org.eclipse.jdt.core/formatter/org/eclipse/jdt/internal/formatter/TextEditsBuilder.java

```diff
@@ -122,12 +125,11 @@ protected boolean token(Token token, int index) {
 
 		bufferWhitespaceBefore(token, index);
 
-		if (token.tokenType == TokenNameCOMMENT_MARKDOWN) {
+		if (token.tokenType == TokenNameCOMMENT_MARKDOWN && !this.options.comment_format_markdown_comment) {
```

Comment:
this is not a good place to disable markdown processing (this would perform all the unneeded processing and only ignore the results at the very end), better change `formattingEnabled` in `CommentsPreparator.visit(Javadoc)`